nntrainer/layers/fc_layer.cpp

```diff
@@ -102,6 +105,46 @@ void FullyConnectedLayer::finalize(InitLayerContext &context) {
       context.requestWeight(bias_dim, bias_initializer, WeightRegularizer::NONE,
                             1.0f, bias_decay, "bias", true);
   }
+
+  /** create weights for LoRA */
+  if (lora_rank) {
+
+    /** loraA : (in_dim.width, lora_rank) */
+    TensorDim loraA_dim(
+      1, is_nchw ? 1 : lora_rank, is_nchw ? in_dim.width() : 1,
+      is_nchw ? lora_rank : in_dim.channel(),
+      TensorDim::TensorType(context.getFormat(), context.getWeightDataType()),
+      is_nchw ? 0b0011 : 0b0101);
+
+    /** loraB: (lora_rank, out_dim) */
+    TensorDim loraB_dim(
+      1, is_nchw ? 1 : unit, is_nchw ? lora_rank : 1,
+      is_nchw ? unit : in_dim.channel(),
```

Comment:
so there's no use of `lora_rank` in the channels last case for B?

Reply:
It should be fixed! Thanks!!:)